hydromt/components/region.py

```diff
@@ -284,12 +286,6 @@ def _parse_region(
     }
     kind = next(iter(kwargs))  # first key of region
     value0 = kwargs.pop(kind)
-    from hydromt.models import MODELS
-
-    if kind in MODELS:
-        model_class = MODELS.load(kind)
-        kwargs = dict(mod=model_class.__init__(root=value0, mode="r", logger=logger))
-        kind = "model"
```

Comment:
I removed it while working on the plugin infrastructure because it was causing circular imports and then forgot to add it back in. I will work on adding this back in